database/buildSchema/18_application_stage_history.sql

```diff
@@ -1,11 +1,9 @@
 -- application stage history
 
-CREATE TYPE public.application_stage AS ENUM ('Screening', 'Assessment', 'Final Decision'); 
-
 CREATE TABLE public.application_stage_history (
     id serial primary key,
     application_id integer references public.application(id),
-    stage public.application_stage,
+    stage_id integer references public.template_stage(id),
     time_created timestamp with time zone,
```

Comment:
Shouldn't it just be `timestamp`, not `timestamp with time zone`?
(I realise this isn't actually a change in this PR, but just noticed it -- should be just `timestamp` throughout, right?)

Reply:
Yea. No problem. I'll fix this.